osu.Game/Users/User.cs

```diff
@@ -203,6 +203,21 @@ public class UserAchievement
             public int ID;
         }
 
+        [JsonProperty("monthly_playcounts")]
```

Comment:
@nanaya how am I supposed to be reading the osu-web docs? It's extremely inconsistent for `User` to the point that I don't trust the docs to even be telling me the correct json properties.

This maps to `$monthlyPlaycounts`:

https://github.com/ppy/osu-web/blob/c353e7386b3b265ae0ed555d64874afbf52c41d5/app/Models/User.php#L75

The one below maps to `$replaysWatchedCounts`:

https://github.com/ppy/osu-web/blob/c353e7386b3b265ae0ed555d64874afbf52c41d5/app/Models/User.php#L97

The `rankHistory` one above maps to `$rankHistories`:

https://github.com/ppy/osu-web/blob/c353e7386b3b265ae0ed555d64874afbf52c41d5/app/Models/User.php#L93

I know that we should be using lower_snake_case for all json properties, but given that there already is the inconsistency in `rankHistory`, it's very hard to know whether these are correct being snake-case themselves...

Reply:
That's because it's not the documentation for json response...

(try checking UserTransformer in Transformers directory instead)

Reply:
I see... Thanks. I guess I should assume `rankHistory` is wrong then since it's not included in any shape/form in there?

Reply:
looks like that one missed the snake case cleanup a while back because of the way it's currently added to the response.